### ISSUE TITLE
I can't see how to make Brave the default browser on Windows

### ISSUE THREAD

**Issue by herby** [brave/browser-laptop#429]

**Comment by bbondy** [comment]
This is a pending task, it's high priority, thanks for posting.

**Comment by bbondy** [comment]
Need something like this:
```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\SOFTWARE\Classes\BraveHTML]
@="Chrome HTML Document"

[HKEY_CURRENT_USER\SOFTWARE\Classes\BraveHTML\DefaultIcon]
@="C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe,0"

[HKEY_CURRENT_USER\SOFTWARE\Classes\BraveHTML\shell]

[HKEY_CURRENT_USER\SOFTWARE\Classes\BraveHTML\shell\open]

[HKEY_CURRENT_USER\SOFTWARE\Classes\BraveHTML\shell\open\command]
@="\"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe\" -- \"%1\""

[HKEY_LOCAL_MACHINE\SOFTWARE\RegisteredApplications]
"Brave"="Software\\Clients\\StartMenuInternet\\Brave\\Capabilities"






[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave]
@="Brave"

[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\Capabilities]
"ApplicationDescription"="Brave is a web browser that runs webpages and applications with lightning speed. It's fast, stable, and easy to use. Browse the web more safely with malware and phishing protection built into Brave."
"ApplicationIcon"="C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe,0"
"ApplicationName"="Brave"

[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\Capabilities\FileAssociations]
".htm"="BraveHTML"
".html"="BraveHTML"
".shtml"="BraveHTML"
".xht"="BraveHTML"
".xhtml"="BraveHTML"
".webp"="BraveHTML"

[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\Capabilities\Startmenu]
"StartMenuInternet"="Brave"

[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\Capabilities\URLAssociations]
"ftp"="BraveHTML"
"http"="BraveHTML"
"https"="BraveHTML"
"irc"="BraveHTML"
"mailto"="BraveHTML"
"mms"="BraveHTML"
"news"="BraveHTML"
"nntp"="BraveHTML"
"sms"="BraveHTML"
"smsto"="BraveHTML"
"tel"="BraveHTML"
"urn"="BraveHTML"
"webcal"="BraveHTML"

;[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\DefaultIcon]
;@="C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe,0"

[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\InstallInfo]
"IconsVisible"=dword:00000001
;"ReinstallCommand"="\"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe\" --make-default-browser"
;"HideIconsCommand"="\"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe\" --hide-icons"
;"ShowIconsCommand"="\"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe\" --show-icons"

[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\shell]

[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\shell\open]

[HKEY_LOCAL_MACHINE\SOFTWARE\Clients\StartMenuInternet\Brave\shell\open\command]
;@="\"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe\""
```

I think the HKLM Stuff can be replaced with HKCU for Win8 and above.  Ideally we'll use Squirrel updater to get it working on win8 and up and then have a separate thing that requires elevation after that.

**Comment by zvezdaburya** [comment]
Encountered the same problem in Ubuntu 14.04 using brave 0.8.3.

I took a look at  ~/.local/share/applications/mimeapps.list and found that brave was not added under [default applications] so I added it manually, basically copypasting the lines which ended in firefox.desktop and replacing firefox.desktop with brave.desktop. 

Now brave shows up in default applications and can be selected, however when clicking a url in a program, say telegram, the browser launches with a blank window. If the browser is already running nothing happens, no new window, tab or anything. That might be my own mistake though, I am sure I have missed a step somewhere...
